src/components/TokenIcon/ModeratedNftIcon.tsx

```diff
@@ -39,10 +39,16 @@ function BlockedNftIcon() {
 }
 
 function ApprovedNftIcon({image}: {image?: string}) {
+  if (typeof image === 'undefined') {
```

Comment:
maybe branch the logic earlier? or even a wrapper to branch the placeholder earlier `<WrapperNFTIcon type: 'approved' | 'consent'> ...consent ...approved or placeholder`